weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1844,7 +1865,7 @@ def _ch_obj_to_obj_schema(ch_obj: SelectableCHObjSchema) -> tsi.ObjSchema:
         digest=ch_obj.digest,
         kind=ch_obj.kind,
         base_object_class=ch_obj.base_object_class,
-        val=json.loads(ch_obj.val_dump),
+        val=json.loads(ch_obj.val_dump) if ch_obj.val_dump else None,
```

Comment:
this makes me nervous that there are other places we blindly assume `val` is jsonable @tssweeney

Reply:
This would be a bug - `val` should never be None. Let's figure out when this is the case

Reply:
correction, `ch_obj.val_dump`

Reply:
This pr would change that to allow None though right? Isn't that what you meant by correctly typing this? SelectableObject would allow an optional val_dump